digitransit-component/packages/digitransit-component-datetimepicker/src/helpers/Datetimepicker.js

```diff
@@ -89,7 +89,14 @@ function Datetimepicker({
   const openPickerRef = useRef();
   const inputRef = useRef();
   const alertRef = useRef();
+  const [invalidInput, setinvalidInput] = useState(false);
 
+  useEffect(() => {
+    document.body.style.setProperty(
+      '--input-color',
+      invalidInput ? '#ff0000' : ' #000000',
+    );
+  }, [invalidInput]);
```

Comment:
This styling looks very unusual. Normally we set styling by combining suitable classes in render method. Is there a reason for this solution?  

Also, it is strange that invalid property is sent from actual timepicker to this parent component with a callback, and the given value is passed back to the sender.  Why not to detect invalidity in timepicker and render a some additional style which gives red input color there?